src/log.cpp

```diff
@@ -21,6 +21,19 @@ spdlog::level::level_enum get_log_level() {
 
 static auto console = spdlog::stdout_color_mt(APP_NAME);
 
+int set_log_level(std::string level) {
```

Comment:
what's the relationship with log_level_set？ log_level_set will read env variable

Reply:
use argument to pass the log level config. or should we just use env variable for log level config?